utils/gitlab_api.py

```diff
@@ -114,6 +124,21 @@ def add_project_member(self, repo_url, user):
             member = project.members.get(user.id)
             member.access_level = gitlab.MAINTAINER_ACCESS
 
+    def add_group_member(self, group_name, user):
+        group = self.gl.groups.get(group_name)
+        try:
+            group.members.create({
+                'user_id': user.id,
+                'access_level': gitlab.MAINTAINER_ACCESS 
```

Comment:
maybe this should be a part of the schema? the group membership would also define the access level. wdyt?

Reply:
hmm. Which schema would it go in? Would each user have access level in their schemas that applies for all groups? Or would the access level be in the gitlab instance schema, so each managed group has an access level? Or I guess the best way to do it would be creating permissions for different access levels for each group. 

Also this could be something the integration could reconcile, for example in app-interface if a user has a permission with owner access  but on gitlab they are just a maintainer the integration could change their access to owner. Not sure how important access levels are to the way you guys manage groups if this is worthwhile or not.

Reply:
its probably not super important, but rather easy to add, and would be really good to have for future use. i'd say it should go in the `gitlab-group-membership` permission schema object. wdyt?